### PR TITLE
dockerize razor-go

### PR DESCRIPTION
**Pull request by OxHimanshu**
#57

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    apt-get install -y nodejs && \
+    npm install --silent && \
+    mkdir tmp/ && cd tmp/ && wget https://golang.org/dl/go1.16.6.linux-amd64.tar.gz && \
+    tar -xvf go1.16.6.linux-amd64.tar.gz && \
+    mv -f go ../../../usr/local && pwd && \
+    cd ../ && rm -rf tmp && \
+    go version && go get -u github.com/ethereum/go-ethereum && \
+    cd ../../go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.6 && chmod +777 build &&  make && make devtools
+RUN npm run
+#RUN ./razor setconfig --provider https://infura/v3/matic --gasmultiplier 1.5
```
Comment (ashish10677) [inline Dockerfile:35]: @SkandaBhat should we run setconfig in the docker image or not?

Comment (SkandaBhat) [reply Dockerfile:35]: Ya we should be able to. This is a user facing feature, and they should have to be able to set config.

Comment (ashish10677) [reply Dockerfile:35]: So this line should be uncommented right?

Comment (SkandaBhat) [reply Dockerfile:35]: yes

**Pull request merged**